### PR TITLE
Updated "Get started with WebView2 in WinUI 2 (UWP) apps"

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered articles for review:
* "Get started with WebView2 in WinUI 2 (UWP) apps"
   * https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2?branch=pr-en-us-2456 [[github rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/winui2-winrt/microsoft-edge/webview2/get-started/winui2.md)]
* "Call native-side WinRT code from web-side code" (WinUI 2 tab content)
   * https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/winrt-from-js?branch=pr-en-us-2456 [[github rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/winui2-winrt/microsoft-edge/webview2/how-to/winrt-from-js.md)]

Phase 1: Changes made in this PR in GetStart article:
* Resolved todo items about updating from 2.7.1 to 2.8 GA.
* Updated steps.
* Updated all screenshots.

Phase 2:
* Related: The finished GetStart WinUI 2 project/solution will be added to the WebView2Samples repo for the first time: https://github.com/MicrosoftEdge/WebView2Samples/pull/172

Phase 3:
* Updated the article [Call native-side WinRT code from web-side code](https://learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/winrt-from-js?tabs=winui2%2Cwinrtcsharp) (WinUI 2 tab), using the updated GetStart article's resulting project as a starting point.

AB#43454995

### CODE REVIEW

File: microsoft-edge/webview2/how-to/winrt-from-js.md [hunk 1]
```diff
@@ -657,9 +677,9 @@ Next, pass the WinRT object from the native side of the host app to the web side
 
 ##### [WinUI 2 (UWP)](#tab/winui2)
 
-1. In Solution Explorer, expand the main project, such as **webview2_sample_uwp**, and then select **MainPage.xaml.cs**.
+1. In Solution Explorer, expand the main project, such as **MyUWPGetStartApp**, expand **MainPage.xaml**, and then select **MainPage.xaml.cs**.
 
-1. Below the `MainPage` constructor, add the following `InitializeWebView2Async` method:
+1. Below the `MainPage(){...}` constructor, add the following `InitializeWebView2Async` method:
```
Comment (lflores-ms) [inline microsoft-edge/webview2/how-to/winrt-from-js.md:682]: Is this change required?

File: microsoft-edge/webview2/get-started/get-started.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ ms.author: msedgedevrel
...
+ms.date: 02/22/2023
 ---
 # Getting Started tutorials
 
 These articles cover how to set up your development tools and create an initial WebView2 app, like a Hello World app with basic functionality.  Learn about WebView2 concepts along the way.
 
 | Article | Completed project | Description |
 |---|---|---|
-| [Get started with WebView2 in WinUI 2 (UWP) apps](winui2.md) | n/a<!--todo: upload finished project--> | Use **C# Blank App (Universal Windows)** project template and install the **Microsoft.UI.Xaml** package (WinUI 2), which installs the **Microsoft.Web.WebView2** SDK package as a dependency. |
+| [Get started with WebView2 in WinUI 2 (UWP) apps](winui2.md) | [WinUI2_GettingStarted](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI2_GettingStarted) | Use **C# Blank App (Universal Windows)** project template and install the **Microsoft.UI.Xaml** package (WinUI 2), which installs the **Microsoft.Web.WebView2** SDK package as a dependency. |
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/webview2/get-started/get-started.md:17]: This link is predicted to work after PR 172 is merged.  
PR 172 status: Approved and ready to merge.
Here is a sibling, comparable link that works now:
https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI3_GettingStarted
The link in question is:
https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI2_GettingStarted

Evidence that the link will work after 172 is merged:
Working branch for PR 172:
https://github.com/mikehoffms/WebView2Samples/tree/user/mikehoffms/getstart-winui2-initial/GettingStartedGuides/WinUI2_GettingStarted
The parent dir for that, shows 3 & 2 are parallel:
https://github.com/mikehoffms/WebView2Samples/tree/user/mikehoffms/getstart-winui2-initial/GettingStartedGuides
with 3 being:
https://github.com/mikehoffms/WebView2Samples/tree/user/mikehoffms/getstart-winui2-initial/GettingStartedGuides/WinUI3_GettingStarted
which matches the 2 link.

File: microsoft-edge/webview2/how-to/winrt-from-js.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/winrt-from-js.md:682]: The change is not technically required, and is intended to add clarity without adding confusion.

Here is the code listing in PR 172, that shows the braces in question, the brace which terminates the ctor:

https://github.com/mikehoffms/WebView2Samples/blob/user/mikehoffms/getstart-winui2-initial/GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/MainPage.xaml.cs

Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/winrt-from-js.md:682]: @peiche-jessica - keep or remove?:   (){...}

File: microsoft-edge/webview2/get-started/get-started.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/webview2/get-started/get-started.md:17]: https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI2_GettingStarted

That link now works, since PR 172 was merged.  @lflores-ms

File: microsoft-edge/webview2/how-to/winrt-from-js.md (hunk 1 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/how-to/winrt-from-js.md:682]: It should be fine without the `(){...}` as there is only one constructor in the code.

Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/winrt-from-js.md:682]: removed change

**Comment by mikehoffms** [comment]
There's a PR UX bug.  Add'l reviews aren't actually required.
